### PR TITLE
[Elixir] Updated Readme

### PR DESCRIPTION
**Pull request by chriseyre2000**
I have added a few extra concepts.

### CODE REVIEW

File: languages/elixir/reference/README.md [hunk 1]
```diff
@@ -27,36 +27,49 @@ Below are the concepts that have been identified in Elixir for use in Exercism.
...
 ### Modules
 
 - Naming
 - Attributes
+  - Redefining
 - Alias
 - Require
 - Import
 - Use
+- Dynamic
```
Comment (Cohen-Carlisle) [inline languages/elixir/reference/README.md]: What does dynamic mean here regarding modules? The only thing that really comes to mind for me is dependency injection of the module, but I'm not sure that's really a concept unto itself.

File: languages/elixir/reference/README.md [hunk 2]
```diff
@@ -27,36 +27,49 @@ Below are the concepts that have been identified in Elixir for use in Exercism.
...
+  - functions prefixed with `do_`
+- Argument positioning conventions
 - [Default arguments](../../../reference/concepts/default_arguments.md)
 - Capture Syntax
+- Guards and defguard
+- Parameters prefixed with `_`
+- Local variables
+- Implicit return
+- Expression results
+- Recursion
```
Comment (neenjaw) [inline languages/elixir/reference/README.md]:
```suggestion
- Multi-clause functions

```

File: languages/elixir/reference/README.md [hunk 3]
```diff
@@ -125,6 +141,7 @@ Below are the concepts that have been identified in Elixir for use in Exercism.
 
 - Let it crash
 - Try, Catch, Rescue
+- {:ok, result}, {:error, reason}
```
Comment (neenjaw) [inline languages/elixir/reference/README.md]:
```suggestion
- Monadic return handling
  - {:ok, result}, {:error, reason}
```

Comment (neenjaw) [reply languages/elixir/reference/README.md]: Or maybe 'Monadic return wrapping' or something like that?

**Comment by chriseyre2000** [comment]
I was trying to avoid using the word monad...Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: Tim Austin <notifications@github.com> Date: 10/03/2020  15:58  (GMT+00:00) To: exercism/v3 <v3@noreply.github.com> Cc: Christopher Eyre <chriseyre2000@gmail.com>, Mention <mention@noreply.github.com> Subject: Re: [exercism/v3] [Elixir] Updated Readme (#855) @neenjaw commented on this pull request.



In languages/elixir/reference/README.md:
> @@ -125,6 +141,7 @@ Below are the concepts that have been identified in Elixir for use in Exercism.
 
 - Let it crash
 - Try, Catch, Rescue
+- {:ok, result}, {:error, reason}

Or maybe 'Monadic return wrapping' or something like that?

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/exercism/v3/pull/855?email_source=notifications\u0026email_token=AAIMRX2OYM5QHGHJB3P72Q3RGZPRHA5CNFSM4K7TT3H2YY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCYW2PTQ#discussion_r390423569",
"url": "https://github.com/exercism/v3/pull/855?email_source=notifications\u0026email_token=AAIMRX2OYM5QHGHJB3P72Q3RGZPRHA5CNFSM4K7TT3H2YY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCYW2PTQ#discussion_r390423569",
"name": "View Pull Request"
},
"description": "View this Pull Request on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by neenjaw** [comment]
@chriseyre2000 That's fair, it's somewhat of a loaded term.  Is there a more erlang/elixir way to word it? Maybe just `Return value wrapping`?

File: languages/elixir/reference/README.md (hunk 3 above)
Comment (neenjaw) [reply languages/elixir/reference/README.md]: Or just `Return value wrapping`?

**Comment by neenjaw** [comment]
Tried using the new web IDE to resolve the merge conflict and it seems to not like our CI.

**Comment by Cohen-Carlisle** [comment]
Thanks for starting this @chriseyre2000 ! Due to the way CI is running some tests, we are forking this over to https://github.com/exercism/v3/pull/912 to resolve the conflicts on rebase encountered during CI.
Superseded by #912

**Comment by ErikSchierboom** [comment]
I've seen this before, and the way I solved it is just by using the command-line:

```
git checkout <pr-branch>
git fetch origin
git rebase origin/master
```

That assumes that origin points to this repository, not to your fork.

**Pull request closed without merging**